src/Pages/HomePage.jsx

```diff
@@ -27,215 +20,173 @@ const HomePage = () => {
+            <PrimaryButton text={"How It Works"} href={"#how-it-works"} />
+            <PrimaryButton text={"FAQs"} href={"#faq"} />
+            <PrimaryButton text={"Why Vacate"} href={"#why-vacate"} />
+          </div>
         </div>
-      </div>
 
-      <div className="row dark-bg" id="how-it-works">
-        <Text variant={"h3"} text={"How It Works"}></Text>
-
-        <div className="col-sm">
-          <img className="icons" src={teamwork} alt={"Teamwork Icon"} />
-          <br />
-          <Text
-            variant={"h5"}
-            text={"We break down the laws into understandable language."}
-            fontFamily={"Lora"}
-          ></Text>
-        </div>
+        <div className="row dark-bg" id="how-it-works">
+          <Text variant={"h3"} text={"How It Works"}></Text>
+
+          <div className="col-sm">
+            <img className="icons" src={teamwork} alt={"Teamwork Icon"} />
+            <br />
+            <Text
+              variant={"h5"}
+              text={"We break down the laws into understandable language."}
+              fontFamily={"Lora"}
+            ></Text>
+          </div>
 
-        <div className="col-sm">
-          <img className="icons" src={phone} alt={"Mobile Phone Icon"} />
-          <br />
-          <Text
-            variant={"h5"}
-            text={"You answer a few simple yes/no questions."}
-            fontFamily={"Lora"}
-          ></Text>
-        </div>
-        <div className="col-sm">
-          <img className="icons" src={employment} alt={"Employment Icon"} />
-          <br />
-          <Text
-            variant={"h5"}
-            text={"This helps determine your vacation eligibility."}
-            fontFamily={"Lora"}
-          ></Text>
+          <div className="col-sm">
+            <img className="icons" src={phone} alt={"Mobile Phone Icon"} />
+            <br />
+            <Text
+              variant={"h5"}
+              text={"You answer a few simple yes/no questions."}
+              fontFamily={"Lora"}
+            ></Text>
+          </div>
+          <div className="col-sm">
+            <img className="icons" src={employment} alt={"Employment Icon"} />
+            <br />
+            <Text
+              variant={"h5"}
+              text={"This helps determine your vacation eligibility."}
+              fontFamily={"Lora"}
+            ></Text>
+          </div>
+          <Text variant={"h3"} text={"Coming in January 2022!"}></Text>
         </div>
-        <Text variant={"h3"} text={"Coming in January 2022!"}></Text>
-      </div>
 
-      <div className="row light-bg" id="faq">
-        <div className="col-sm">
-          <Text variant="h3" text={"FAQs"}></Text>
-          <div id="accordion">
-            <FAQAccordion />
+        <div className="row light-bg" id="faq">
+          <div className="col-sm">
+            <Text variant="h3" text={"FAQs"}></Text>
+            <div id="accordion">
+              <FAQAccordion />
```

Comment:
Did the styling of the FAQ accordion change? It's showing up as all white on my end.

Reply:
I think it has to do with having both mui v4 and v5 at the same time. I fixed it but we should consider removing v5 unless we are going back to React 17 like on the TS branch.